manager/dispatcher/assignments.go

```diff
@@ -245,3 +260,24 @@ func (a *assignmentSet) message() api.AssignmentsMessage {
 
 	return message
 }
+
+// populateSecretFromPlugin populates the secret value for the given specification using the secret plugin subsystem.
+func (a *assignmentSet) populateSecretFromDriver(spec *api.SecretSpec, readTx store.ReadTx) error {
```

Comment:
This function still confuses me a bit. How is the secret populated for the built-in raft store?